pymongo/ssl_support.py

```diff
@@ -56,20 +56,18 @@
 
     if HAVE_PYSSL:
         PYSSLError: Any = _pyssl.SSLError
-        BLOCKING_IO_ERRORS: tuple = _pyssl.BLOCKING_IO_ERRORS + _ssl.BLOCKING_IO_ERRORS
-        BLOCKING_IO_READ_ERROR: tuple = (
-            _pyssl.BLOCKING_IO_READ_ERROR,
-            _ssl.BLOCKING_IO_READ_ERROR,
-        )
-        BLOCKING_IO_WRITE_ERROR: tuple = (
-            _pyssl.BLOCKING_IO_WRITE_ERROR,
-            _ssl.BLOCKING_IO_WRITE_ERROR,
-        )
+        BLOCKING_IO_ERRORS: tuple = _ssl.BLOCKING_IO_ERRORS + _pyssl.BLOCKING_IO_ERRORS
+        BLOCKING_IO_READ_ERROR: Any = Union[
```

Comment:
We cannot use Union at runtime. These should also be tuples.

Reply:
Yeah (see convo chain above HAHA) they're back to tuples now :)